migrations/sql/afterMigrate__87__create_test_db.sql

```diff
@@ -1,13 +0,0 @@
-DO
-$$
-DECLARE
-BEGIN
-  CREATE EXTENSION IF NOT EXISTS dblink;
-  IF (EXISTS (SELECT 1 FROM pg_database WHERE datname = 'mds_test')) THEN
-    RAISE NOTICE 'Database already exists';
-  ELSE
-    PERFORM dblink_connect('');
-    PERFORM dblink_exec('CREATE DATABASE mds_test OWNER mds');
```

Comment:
Commenting out these two lines make the migration a success, so I suppose there is no need for this file.

Reply:
This is necessary for setting up `test` database locally. I think migration will fail otherwise!

Reply:
does local testing run against test database?

Reply:
I believe so. we would like to run integration tests locally to make sure they pass locally before pushing it for a pull request

Reply:
We use mds_test to run integration tests locally as Hitankar mentioned so we should keep this.